packages/components/Button/src/ToggleButton/__snapshots__/ToggleButton.test.tsx.snap

```diff
@@ -37,13 +37,13 @@ exports[`ToggleButton default 1`] = `
       "backgroundColor": "#f3f2f1",
       "borderColor": "#8a8886",
       "borderRadius": 4,
-      "borderWidth": 1,
+      "borderWidth": 1.5,
       "display": "flex",
       "flexDirection": "row",
       "justifyContent": "center",
```

Comment:
I don't think the tests were ever actually running on win32? I think it's an open question still which platform each control should run it's test on.. ideally each component would just have tests for all platforms, which I think is possible but would slow us down 5x 😅 

Probably worth a follow-up. I think the answer changes for each component.

Reply:
This is the first time a jest test is running on win32. Most jest tests have been running on Android (with the exception of some tests recently getting switched over to iOS). It might look like the tests were running on win32/windows because up until recently iOS was using windows global tokens

Reply:
yup, I do think that the tests were still having more of a win32/windows coverage because of the tokens being same in both android and iOS till recently. Maybe actually having tests running for each endpoint that validates snapshots and endpoint specific unit tests would be a good coverage, but if its possible and slows down inner loop then it's a weighted choice. Taking it more in offline chat.